cmd_run.go

```diff
@@ -242,12 +262,13 @@ func RunAction(c *cli.Context) error {
 }
 
 func (t *tiros) InitRun(c *cli.Context) (*models.Run, error) {
-	version, sha, err := t.kubo.Version()
+	version, sha, err := t.ipfs.Version()
 	if err != nil {
-		return nil, fmt.Errorf("kubo api offline: %w", err)
+		return nil, fmt.Errorf("ipfs api offline: %w", err)
 	}
 
-	dbRun, err := t.dbClient.InsertRun(c, fmt.Sprintf("%s-%s", version, sha))
+	ipfsImpl := c.String("ipfs-implementation")
+	dbRun, err := t.dbClient.InsertRun(c, ipfsImpl, fmt.Sprintf("%s-%s", version, sha))
```

Comment:
FYI @dennis-tra this is failing for me locally saying `ERRO[0000] error: init run: ipfs api offline: version: command not found`. Is this trying to call 'ipfs' binary locally to return version?

Reply:
It's not using the local Kubo binary. It's using the HTTP API exposed by Kubo. In this case, the must be an endpoint that implements this specification:  https://docs.ipfs.tech/reference/kubo/rpc/#api-v0-version